packages/buildpacks/src/push.ts

```diff
@@ -0,0 +1,3 @@
+module.exports = function (remote?: string) {
+  return `git push ${remote || 'heroku'} master`
```

Comment:
what is this for?

Reply:
It comes from the existing `buildpacks` namespace we inherited. It's used to return the right remote apparently when messaging, see [this](https://github.com/heroku/cli/pull/1000/commits/6f9dfc57cb836dc4ed87475ee8f026dbb120b134#diff-c51a8022b812853019836134ceaebff5R132).